libnavui-voice/src/main/java/com/mapbox/navigation/ui/voice/api/MapboxAudioGuidance.kt

```diff
@@ -147,7 +148,8 @@ internal constructor(
                         .filter { it.voiceInstructions != lastPlayedInstructions }
                         .flatMapConcat {
                             lastPlayedInstructions = it.voiceInstructions
-                            audioGuidance.speak(it.voiceInstructions)
+                            val announcement = audioGuidance.speak(it.voiceInstructions)
+                            flowOf(announcement)
```

Comment:
Since you emit a flow with a single value, you can use a simple `map` instead if `flatMapConcat`. 
After that you can replace two consecutive `map`s with one `map`.